ipatests/test_integration/test_commands.py

```diff
@@ -187,3 +194,61 @@ def test_list_help_topics(self):
             raiseonerr=False
         )
         assert result.returncode == 0
+
+    def test_ssh_key_connection(self):
+        """
+        Integration test for https://pagure.io/SSSD/sssd/issue/3747
+        """
+
+        test_user = 'test-ssh'
+        master = self.master.hostname
+
+        with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
I think you can use pytest's [tmpdir](https://docs.pytest.org/en/2.7.3/tmpdir.html), it supports Python 2.

Reply:
@netoarmando Thanks for checking this. Yep, changed. Lets wait for the tests...